setup.cfg

```diff
@@ -28,6 +28,7 @@ classifiers =
     Programming Language :: Python :: 3 :: Only
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
```

Comment:
All changes made in this file will not be saved. Please revert this and move it to the `setup_config/base.cfg`file. _(correctly)_